config/datatypes_conf.xml.sample

```diff
@@ -443,8 +443,10 @@
     <sniffer type="galaxy.datatypes.tabular:Vcf"/>
     <sniffer type="galaxy.datatypes.binary:TwoBit"/>
     <sniffer type="galaxy.datatypes.binary:GeminiSQLite"/>
-    <sniffer type="galaxy.datatypes.proteomics:MzSQlite"/>
+    <sniffer type="galaxy.datatypes.binary:MzSQlite"/>
```

Comment:
What would it break? It seems right now MzSQlite can't be sniffed at all because it's not being registered as a binary sniffer. I can roll back the MzSQLite changes if necessary though.